ci-operator/step-registry/openshift-qe/installer/aws/sdn/fips/OWNERS

```diff
@@ -0,0 +1 @@
+../../OWNERS
```

Comment:
this owners file should refernce back to the OWNERS file [here](https://github.com/openshift/release/blob/af37b5aa8a09837fd5d65208255caa619149442d/ci-operator/step-registry/openshift-qe/OWNERS) 

Think it needs 4 "../"